Automattic/jetpack
# Boost: Fix prettier configuration and fix linting issues

#### Changes proposed in this Pull Request:

* Transform the `.prettierrc` config file to its JavaScript version `.prettierc.js`.
* Extend Boost `.prettierc.js` file with the prettier config from the root of the repo.
* Fix linting issues.

#### Jetpack product discussion


N/A

#### Does this pull request change what data or activity we track or use?


N/A

#### Testing instructions:






* Inside the boost plugin root folder run `pnpm run lint:js` and make sure linting is successful.
